### PR TITLE
Create updated tests for APG design pattern example: Editable Combobox With Both List and Inline Autocomplete

### LINKED ISSUE #340: Create updated tests for APG design pattern example: Editable Combobox With Both List and Inline Autocomplete
#### Applies To

* APG design pattern: [Combobox](https://w3c.github.io/aria-practices/#combobox)
* Specific example: [Editable Combobox With Both List and Inline Autocomplete](https://w3c.github.io/aria-practices/examples/combobox/combobox-autocomplete-both.html)

#### Testing Notes

This issue thread will house all documentation relating to the development of a test plan for the "Editable Combobox With Both List and Inline Autocomplete" APG example, which conforms to the "Combobox" design pattern.  All follow-ups (including the test plan itself) will be added as new comments to facilitate notifications, but this initial comment will be updated with relevant details and URLs as needed for convenience.

#### Additional References

* [combobox (role)](https://w3c.github.io/aria/#combobox)
* [listbox (role)](https://w3c.github.io/aria/#listbox)
* [Option (role)](https://w3c.github.io/aria/#option)
* [aria-activedescendant (property)](https://w3c.github.io/aria/#aria-activedescendant)
* [aria-controls (property)](https://w3c.github.io/aria/#aria-controls)
* [aria-label (property)](https://w3c.github.io/aria/#aria-label)
* [aria-expanded (state)](https://w3c.github.io/aria/#aria-expanded)
* [aria-selected (state)](https://w3c.github.io/aria/#aria-selected)

### PR DESCRIPTION
**Pull request by jscholes**
[Preview Tests](https://raw.githack.com/w3c/aria-at/tests/combobox-autocomplete-both-updated/build/index.html)

Resolves #340

### CODE REVIEW

**Comment by jscholes** [comment]
@rmeritz The GitHub Action to generate test files automatically doesn't seem to be running for this PR.  There were some errors in the CSV files, which I detected by running generation locally and then fixed.  But I can't even see any failures server-side; the script just doesn't seem to be running full-stop.

Could you or someone else on the team please investigate?  We ideally need the generated preview for internal review.

**Comment by mcking65** [comment]
Definitely a problem with preview. I see 30 tests in the [test plan review](https://raw.githack.com/w3c/aria-at/tests/combobox-autocomplete-both-updated/review/combobox-autocomplete-both.html) but 76 tests in the csv file.

**Comment by jscholes** [comment]
@mcking65 For reference, our updated tests are in a separate directory with an "-updated" suffix.  So once the preview has been generated, this pattern will have two entries in the table, one with 30 tests and one with 76 tests.  We can then discuss removing or merging as appropriate.

I can generate the test files locally and push, if that would help?  Obviously we need to know what's gone wrong with the GitHub action as well.

**Comment by rmeritz** [comment]
@jscholes - I will look into and work on a fix for this today.

**Comment by jscholes** [comment]
@rmeritz Great, thanks!

**Comment by rmeritz** [comment]
@jscholes - I spent a bit of time today trying to figure out why the github action is no longer getting enqueded. But I wasn't able to quickly figure out the issue, I'll continue to try figure out the root cause. But for now, in order not to block your work I think it would be best just to manually generate and commit the files. I've done it for this PR for now.

**Comment by rmeritz** [comment]
@jscholes - @evmiguel figured out the problem with the github action that generates and commits the test files. 

Someone had turned if out manually via this dashboard on github: https://github.com/w3c/aria-at/pull/372

They turned it back on and now it is running again.

File: tests/resources/keys.mjs [hunk 1]
```diff
@@ -70,8 +70,10 @@ export const A                                = "A";
 export const B                                = "B";
 export const SHIFT_B                          = "Shift+B";
 export const C                                = "C";
+export const SHIFT_C                                = "Shift+C";
```
Comment (zcorpan) [inline tests/resources/keys.mjs:73]: This causes and error because line 13 already declares `SHIFT_C`

Comment (zcorpan) [reply tests/resources/keys.mjs:73]:
```suggestion
```

**Comment by zcorpan** [comment]
I fixed the duplicate `SHIFT_C` so I could view the tests.

This PR also needs to be rebased and resolve merge conflicts.

**Comment by jscholes** [comment]
@zcorpan Thanks!

> This PR also needs to be rebased

Could you expand on this?

**Comment by zcorpan** [comment]
@jscholes GitHub says "This branch cannot be rebased due to conflicts". This can be resolved by rebasing the branch on current `master`. Though I see that this has merged `master` back into this branch, which is a different way of dealing with conflicts. Personally I prefer rebase, but we don't have a policy on how to resolve conflicts in this repo. 🙂

**Pull request merged**